atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolImpl.java

```diff
@@ -279,7 +279,7 @@ private synchronized void refreshPool() {
         if (config.autoRefreshNodes()) {
             setServersInPoolTo(cassandra.refreshTokenRangesAndGetServers());
         } else {
-            setServersInPoolTo(config.servers());
+            setServersInPoolTo(config.servers().thrift());
```

Comment:
another usage that's kinda awkward, I imagine this is where your code will come into effect?

Reply:
Since this is the existing ClientPool which only recognizes thrift clients. I don't plan to pass anything else other that those endpoints. If this answers your question.